### ISSUE TITLE
Use your code for textual entailment

### ISSUE THREAD

**Issue by mmmjung** [cooelf/SemBERT#7]
Hi, thanks for the code!

Can I somehow use your code to get textual entailment probabilities for pairs of sentences from my own dataset? If yes, please help give me some hints.

**Comment by cooelf** [comment]
You can use the logits here: https://github.com/cooelf/SemBERT/blob/master/run_classifier.py#L1223
